python/tests/test_run_config.py

```diff
@@ -3,7 +3,7 @@
 from aiconfig.Config import AIConfigRuntime
 from mock import patch
 
-from .conftest import mock_openai_chat_completion, set_temporary_env_vars
+from .conftest import mock_openai_chat_completion
```

Comment:
Just confirming it's not needed in the function def below? Not sure if the pytest decorator does magic like calling with this as the param

Reply:
No magic AFAIK. test passes